free-2.md

```diff
@@ -11,34 +11,55 @@ _What need is met by your idea? Who are you helping? What is that person's
+
+Additionally, this process only works on one symbol at a time. In my project, you could do something like:
+
+floor(n over 2)
+
+and get $\lfloor \frac{n}{2} \rfloor$ as a result.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+The need is very simple and domain specific. You need to figure out how to write a symbol in latex, so you ask a language with a simple query syntax and get an answer. Seems self-evident here. 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+First, I came up with this by looking at the reccomended domains section of the project ideas page. Then, I thought about things that I would like out of latex, and the first thing that came to mind was an easier way of finding out how to write what I want. I'm excited by this, because since I write latex a lot, its something that I would actually use. 
 
 ### Domain
 _Describe the project's domain in five words._
 
+Latex symbol syntax lookup helper 
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+This is the most important part of this project. If the language isn't intuitive to use, then its no better than latex, as the user would have to look up in the projects documentation how to write what they want to get what they want in latex. Not very useful. The basic idea would be two things. First, a user could specify a mathematical statement in the syntax of the language, and save it as a variable. The language would output the syntax for this, and then save the variable. One could then modify the original variable, so it would look like:
+var x = [statement in math language]
+> [latex syntax for statement]
+var y = x + 2
+> $[latex syntax for statement x] + 2$
+and as such reduce some of the copy-paste work that goes into latexing math. Obviously this example is trivial since adding two to a statement is easy, but if it were something that involved more complex syntax, this could save a lot of time. 
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+When the program runs, the user's input would be parsed by an engine that then looks for the symbol that best matches what they asked for. This should be relatively simple. The real issue is if the user looks for something that doesn't exist (or mispells something). Instead of just erroring and breaking, the system should try to "fuzzy match", or find something that the user probably meant. It should then ask for the users input here, probably displaying a list of possible alternatives (a la Google's "did you mean X?"). This output would be hard to do via image, but could be done through text pretty easily. A GUI for this could let you do it via image.
 
```

Comment:
It'd be important to make this as fast as possible so it's more convenient than other methods